user_management/src/user/views/send_user_infos.py

```diff
@@ -0,0 +1,51 @@
+from django.core.mail import send_mail
+from django.http import HttpRequest, JsonResponse
+from django.utils.decorators import method_decorator
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+
+from common.src.internal_requests import InternalRequests
+from common.src.jwt_managers import user_authentication
+from user.models import User
+from user.views.forgot_password import anonymize_email
+from user_management import settings
+from user_management.JWTManager import get_user_id
+
+
+@method_decorator(user_authentication(['GET']), name='dispatch')
+@method_decorator(csrf_exempt, name='dispatch')
+class SendUserInfosView(View):
+
+    @staticmethod
+    def get(request: HttpRequest) -> JsonResponse:
+        user_id = get_user_id(request)
+        user = User.objects.get(id=user_id)
+        infos = {'username': user.username,
+                 'email': user.email,
+                 'avatar': settings.USER_MANAGEMENT_URL + f'user/avatar/{user.username}/',
+                 'two_fa': user.has_2fa,
+                 }
+
+        for key, url in {'user_stats_id': settings.USER_STATS_URL + f'statistics/user/{user.id}/',
+                         'user_stats_history': settings.USER_STATS_URL + f'statistics/user/{user.id}/history/',
+                         'user_stats_progress': settings.USER_STATS_URL + f'statistics/user/{user.id}/progress/',
+                         }.items():
+
+            access_token = request.headers.get('Authorization')
```

Comment:
You could move this line out of the for loop so you dont have to retrieve it from the request headers for each iteration